dspace/config/modules/external-providers.cfg

```diff
@@ -25,4 +25,25 @@ vufind.url.search = https://vufind.org/advanced_demo/api/v1/search
 
 scielo.url = https://search.scielo.org/?output=ris&q=
 
-#################################################################
\ No newline at end of file
+#################################################################
+#----------------------   ADS   --------------------------------#
+#---------------------------------------------------------------#
+
+submission.lookup.ads.key =
+
+ads.url = https://api.adsabs.harvard.edu/v1/search/query
+ads.resultFieldList = abstract,ack,aff,alternate_bibcode,alternate_title,arxiv_class,author,bibcode,bibgroup,bibstem,citation_count,copyright,database,doi,doctype,first_author,grant,id,indexstamp,issue,keyword,lang,orcid_pub,orcid_user,orcid_other,page,property,pub,pubdate,read_count,title,vizier,volume,year
+
+#################################################################
+#----------------------   EPO   --------------------------------#
```

Comment:
Same here, in the header we should spell out what EPO stands for.  Event just something as simple as saying: "European Patent Office (EPO)"